aws-test/tests/aws_ec2_ami/test-hydrate-expected.json

```diff
@@ -6,16 +6,16 @@
     "image_id": "{{ output.resource_id.value }}",
     "launch_permissions": [
       {
-        "Group":null,
+        "Group":"",
         "OrganizationArn": null,
         "OrganizationalUnitArn": null,
-        "UserId":"388460667113"
+        "UserId":"013122550996"
```

Comment:
Can we remove this hardcoded data as it will fail during every integration test run?